site/hugo.toml

```diff
@@ -8,7 +8,7 @@ defaultContentLanguageInSubdir = false
 # Useful when translating.
 enableMissingTranslationPlaceholders = true
 
-enableRobotsTXT = true
+enableRobotsTXT = false
```

Comment:
xref: https://gohugo.io/templates/robots/#robotstxt-template-example
```
To create a robots.txt file without using a template:

Set enableRobotsTXT to false in the site configuration.
Create a robots.txt file in the static directory.
Remember that Hugo copies everything in the [static directory](https://gohugo.io/getting-started/directory-structure/) to the root of publishDir (typically public) when you build your site.
```

Reply:
can you add a comment in the file, saying we are manually maintaining it